baseplate/lib/log_formatter.py

```diff
@@ -5,7 +5,13 @@ class CustomJsonFormatter(jsonlogger.JsonFormatter):
     def process_log_record(self, log_record: dict) -> dict:
         log_record["level"] = log_record.pop("levelname", None)
         try:
-            log_record["traceID"] = int(log_record["threadName"])
+            # see if there's an integer thread name, that's probably a traceID
+            # from the logging observer.
+            trace_id = int(log_record["threadName"])
```

Comment:
:no_bell: Why not just make this one line?  Otherwise, LGTM.

Reply:
just so I could add some explanation in comments for why we're doing `str(int(str))`